pootle/apps/accounts/utils.py

```diff
@@ -419,3 +421,34 @@ def update_user_email(user, new_email):
     validate_email(new_email)
     user.email = new_email
     user.save()
+
+
+def get_user_list_with_permission(permission_code, project, language):
```

Comment:
@julen I was changing `list` to `QuerySet` when I got your comment. 
So it's done.